library/src/main/kotlin/display/bridgeSearch/implemetation/GraphBridgeSearchTarjan.kt

```diff
@@ -0,0 +1,58 @@
+package display.bridgeSearch.implementation
+
+import display.bridgeSearch.GraphBridgeSearch
+import graph.Graph
+
+class GraphTarjanBridgeFinder<V, E> {
```

Comment:
класс должен реализовывать интерфейс

```suggestion
class GraphTarjanBridgeFinder<V, E> : GraphBridge {
```